src/containers/navigation-icons/user-icons/UserIcons.tsx

```diff
@@ -35,13 +53,31 @@ const UserIcons: FC<UserIconsProps> = ({ setSidebarOpen }) => {
             openNotifications,
             setSidebarOpen
           })}
-          icon={item.icon}
+          icon={getUserIcon(response) ?? item.icon}
           key={item.tooltip}
           tooltip={t(item.tooltip)}
         />
       )
   )
 
+  function getUserIcon(user: any) {
```

Comment:
Using `any` is very bad practice
```suggestion
  function getUserIcon(user?: User) {
```